clients/cloud/go/README.md

```diff
@@ -0,0 +1,73 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using [Confluent Golang Client for Apache Kafka](https://github.com/confluentinc/confluent-kafka-go).
+
+
+# Prerequisites
+
+* [Confluent's Golang Client for Apache Kafka](https://github.com/confluentinc/confluent-kafka-go) installed on your machine
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+
```

Comment:
The Python client contains librdkafka so it can be installed in the standard way (pip install..).
The go client on the other hand cannot simply be 'go get'inated but requires an os dependent pre install step of librdkafka.

I think we should add a 'make dependencies' to the go client to perform this step, but it still needs to be manually executed

Reply:
I updated the link to https://github.com/confluentinc/confluent-kafka-go#installing-librdkafka